bin/all_sky_search/pycbc_bin_trigger_rates_dq

```diff
@@ -54,14 +54,23 @@ ifo = args.ifo
 # This works as a pre-filter as SNR is always greater than or equal
 # to sngl_ranking, except in the psdvar case, where it could increase.
 
+def check_snr_psdvar(snr, psd_var_val):
+    """
+    Convenience function to calculate the maximum possible ranking statistic
+    value given SNR and psd variation statistic
+    """
+    rw_snr = snr / psd_var_val ** 0.5
+    rw_snr[psd_var_val == 0] = 0
```

Comment:
What does it mean for psd var to be =0?  Ie what could cause that?

What value do we get if it is not computed in the first place?

Reply:
`pycbc_inspiral` does not store it if not computed. PyCBC Live has not yet defined behaviour in this case.

.... If this will be fixed "in a better way" soon anyway, we probably don't need the "is it equal to 0" check (but it won't harm). Note that in the ranking computation, psd_var rescaling is not applied if the value is smaller than 0.65.

Reply:
Noting that the current code would fail if psd_var_val is not computed.

Reply:
Do you want me to prevent that failure? It would be a case of minor changes in the select function, where if the dataset isn't present but is asked for, it could be set to zero?

Reply:
For now (avoiding things that should be in #4524), I would limit changes to this executable, and use a if statement to use the old call to select if psd_var is not present, and the new one if it is.

Reply:
Failure is good if the alternative is continuing while producing a plausible but wrong answer .. 
Just noting that substituting a 'silent' default value might be rather dangerous

Reply:
I think wrapping in a if statement would work here. We aren't giving an option being ignored, we are simply working around a dataset which may or may not have been saved

Reply:
Can we then remove `rw_snr[psd_var_val == 0] = 0` ? If psd var is calculated the psd var val is 0, there is a big problem and I think we want the code to produce an error rather than ignore it ..

Reply:
Please remove this line